webapp/public/js/domjudge.js

```diff
@@ -659,3 +659,14 @@ function initializeAjaxModals()
 		});
 	});
 }
+
+function pinScoreheader(){
+	var sboard = new URL(window.location).searchParams.get("static");
```

Comment:
consider using a more descriptive variable name instead of `sboard`.

Reply:
I tried, I also found a mistake that when one uses capitals it fails so thats now also fixed I think